### PR TITLE
Correct the order of passing arguments to the InputGenerateData constructor (Fix Round, Ceiling Conformance tests for Template plugin)

### PR DESCRIPTION
**Pull request by nsemaev**
### Details:
 - *Correct the order of passing arguments to the InputGenerateData constructor*

### Tickets:
 - *81999*

### CODE REVIEW

File: src/tests/functional/shared_test_classes/src/base/utils/generate_inputs.cpp [hunk 1]
```diff
@@ -63,28 +63,28 @@ ov::runtime::Tensor generate(const std::shared_ptr<ngraph::op::v0::Acos>& node,
                              size_t port,
                              const ov::element::Type& elemType,
                              const ov::Shape& targetShape) {
-    return Activation::generate(elemType, targetShape, InputGenerateData(-1, 2, 32768, 1));
+    return Activation::generate(elemType, targetShape, InputGenerateData(2, -1, 32768, 1));
```
Comment (gkrivor) [inline src/tests/functional/shared_test_classes/src/base/utils/generate_inputs.cpp:66]: I suggest to change definition/declaration of InputGenerateData and change order of range/startfrom to startfrom/range.

Looks like it shouldn't cause any issues: grep shows it uses only in generate_inputs.cpp.

**Comment by nsemaev** [comment]
> > Correcting the range for Random
> 
> @nsemaev please share why do we need these changes and provide examples

I added a picture to the description and gave a comparison

**Comment by nsemaev** [comment]
[report.xml.txt](https://github.com/openvinotoolkit/openvino/files/8328294/report.xml.txt)
Round, Ceiling are passed

File: src/tests/ie_test_utils/common_test_utils/data_utils.hpp [hunk 2]
```diff
@@ -194,14 +194,14 @@ void inline
 fill_data_random(T *pointer, std::size_t size, const uint32_t range = 10, int32_t start_from = 0, const int32_t k = 1,
                  const int seed = 1) {
     testing::internal::Random random(seed);
-    random.Generate(range);
+    random.Generate(range * k);
```
Comment (gkrivor) [inline src/tests/ie_test_utils/common_test_utils/data_utils.hpp:197]: Question for anyone. Just curious: what do we here? Do we skip first random value for some purpose? What is the purpose?

**Review by gkrivor — APPROVED** [review]

**Review by iefode — APPROVED** [review]

**Pull request merged**